extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -252,8 +254,11 @@ class AmpLightbox extends AMP.BaseElement {
     return Promise.resolve();
   }
 
-  /** @override */
-  activate() {
+  /**
+   * @param {?../../../src/service/action-impl.ActionInvocation=} opt_invocation
+   * @private
+   */
+  open_(opt_invocation) {
```

Comment:
Minor: `opt_invocation` is never read. Remove the param and change the default action handler to `() => this.open_()`.